server/api/events.js

```diff
@@ -0,0 +1,74 @@
+const router = require('express').Router()
+const {Event} = require('../db/models')
+module.exports = router
+
+router.get('/', async (req, res, next) => {
+  try {
+    const events = await Event.findAll()
+    res.send(events)
+  } catch (err) {
+    next(err)
+  }
+})
+router.get('/:eventId', async (req, res, next) => {
+  try {
+    const events = await Event.findById(req.params.eventId)
+    res.send(events)
+  } catch (err) {
+    next(err)
+  }
+})
+
+router.get('/name/:eventName', async (req, res, next) => {
```

Comment:
This is cool! I didn't know you could do this! What happens when the name has spaces or weird characters though?

Reply:
We could do the blob thing where we change spaces into underscores